### DISCUSSION TITLE
I can't find SoundSwitch after closing the settings / the GUI doesn't open again

### DISCUSSION THREAD

**Discussion by Belphemur** [Belphemur/SoundSwitch#850] [FAQ]
I can't find the way to get the settings again. Reopening the app doesn't do anything.

**Comment by Belphemur** [comment]
The way SoundSwitch works is in the background. The first time you open the software, SoundSwitch will show you the settings menu.

After that to access SoundSwitch you need to go in the tray icon.
![TrayIcon](https://user-images.githubusercontent.com/197810/146657919-e31afe06-96d6-4253-8e96-6103852eed55.gif)

**Reply by kingkong8986** [reply]
i tested v7.0.0 version is quite good.  However, the problem of opening settings in Win11 Pro, need to close the App in the taskbar icon, relaunch it again, then from icon, able to see Settings again. Every time need to kill the app, relauch it again, then settings will appear.  Strange but works for me.
